schemas/test/en/test_benchmark_business.json

```diff
@@ -2,12 +2,11 @@
     "language": "en",
     "mime_type": "application/json/ons/eq",
     "schema_version": "0.0.1",
-    "data_version": "0.0.1",
-    "survey_id": "017",
-    "form_type": "0070",
-    "theme": "business",
+    "data_version": "0.0.3",
+    "survey_id": "0",
+    "theme": "default",
     "legal_basis": "Notice is given under section 1 of the Statistics of Trade Act 1947.",
-    "title": "Quarterly Stocks Survey",
+    "title": "Benchmark-Survey",
     "questionnaire_flow": {
         "type": "Linear",
```

Comment:
Since we have multiple sections we should use a Hub here to replicate real multi section business surveys?

Reply:
I can see that the unhappy path is adding multiple people, but if this was converted to hub could test journeys where a new section is being opened up on an unhappy path, editing or removing a list item and/or going back to edit a previous section to open up a new path?

Reply:
We have tried using the Hub before this but we found that the benchmark redirects do not work as expected.
The benchmark still is experiencing  this error:
`GET /questionnaire: CatchResponseError('Expected a (302) but got a (200) back when getting page: /questionnaire')`.

Reply:
Updated to Hub 👍